.circleci/install-released-version.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
```

Comment:
This could be a much simpler shell script:

```sh
#!/bin/bash
version=$(jq -r .version ../package.json)
npm init -y
npm install "react-axe@$version"
``1